packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -56,6 +57,60 @@ export default function FeedFilters({
     }
   };
 
+  //   const tabs = [
+  //     {
+  //       title: FilterMenuTitle.Tags,
+  //       options: { icon: <HashtagIcon /> },
+  //     },
+  //     {
+  //       title: FilterMenuTitle.Advanced,
+  //       options: { icon: <FilterIcon /> },
+  //     },
+  //     {
+  //       title: FilterMenuTitle.Blocked,
+  //       options: { icon: <BlockIcon /> },
+  //     },
+  //   ];
+  //   return (
+  //     <Modal
+  //       className="h-full mb-8 flex-1"
```

Comment:
should the h-full not be part of the guidelined modal in this case? (basically all these classes?)

Reply:
Feed filters has a full screen height modal which is not part of the modal guidelines. Thats why I added the classes outside. But maybe we can check with `@ design` to see if it should be part of the guidelines